main.js

```diff
@@ -0,0 +1,85 @@
+// the game board to display with the 1st peg being the "start" peg
+let board = [
+  [5,4,3,2,1],
+  [],
+  []
+]
+
+// function to execute to make the board visible
+function displayBoard() {
+  console.clear()
+  board.map(function(pegs) {
+    console.log(`--- ${pegs}`)
+    return
+  })
+}
+
+
+
+
+// -- main game-play function -- // 
+
+
+function moveDiscs(start, dest) {
+  start--
+  dest--
+
+  // check if the starting peg has any discs to move
+if (board[start].length === 0) {
+  console.log('no discs to move')
+  return
+}
+
+// a disc to move
+const disc = board[start].pop()
```

Comment:
you are not only getting the element that is returned from pop, but modifying the array without a valid movement, that is a VERY bad practice. Do not perform an action unless its valid. I see in line 40 that you had to readd the disc, imagine you forget, big bug.